src/openforms/scss/admin/_app_overrides.scss

```diff
@@ -96,6 +96,42 @@ $djai-padding--tablet: 30px;
   text-align: center;
   clear: both;
   padding-top: 1em;
+
+  &--disabled {
+    display: none;
+  }
```

Comment:
can you please use the mixins like in the sample I gave? These exist because the resulting selector now is `.admin-login-option--disabled`, and with the mixin it is `.admin-login-option.admin-login-option--disabled`.

A classname with modifier may not exist without the block/element itself, these sass mixins help enforce this.

Reply:
I added it for `.admin-login-option--disabled`, though I'm not sure if this works for the `#login-form` block, because it's an id instead of a class :thinking:

Reply:
yeah it doesn't work with IDs, since the whole idea of BEM is class-names :) for those you can spell them out manually entirely. ID's are an anti-pattern anyway for styling, I dislike that Django uses them so much in their CSS :(